[2021-12-23 16:37:46,640] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.echo_meet_date 2021-12-23T16:37:43.368103+00:00 [queued]>
[2021-12-23 16:37:46,652] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.echo_meet_date 2021-12-23T16:37:43.368103+00:00 [queued]>
[2021-12-23 16:37:46,660] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-23 16:37:46,661] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-23 16:37:46,661] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-23 16:37:46,687] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): echo_meet_date> on 2021-12-23T16:37:43.368103+00:00
[2021-12-23 16:37:46,688] {standard_task_runner.py:52} INFO - Started process 347606 to run task
[2021-12-23 16:37:46,703] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'echo_meet_date', '2021-12-23T16:37:43.368103+00:00', '--job-id', '1760', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpltt42p8f', '--error-file', '/tmp/tmp44m4f4p8']
[2021-12-23 16:37:46,705] {standard_task_runner.py:77} INFO - Job 1760: Subtask echo_meet_date
[2021-12-23 16:37:46,784] {logging_mixin.py:104} INFO - Running <TaskInstance: test.echo_meet_date 2021-12-23T16:37:43.368103+00:00 [running]> on host airflowvm
[2021-12-23 16:37:47,017] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=echo_meet_date
AIRFLOW_CTX_EXECUTION_DATE=2021-12-23T16:37:43.368103+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-23T16:37:43.368103+00:00
[2021-12-23 16:37:47,021] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-23 16:37:47,022] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo 211228']
[2021-12-23 16:37:47,028] {subprocess.py:75} INFO - Output:
[2021-12-23 16:37:47,029] {subprocess.py:79} INFO - 211228
[2021-12-23 16:37:47,033] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-23 16:37:47,057] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=test, task_id=echo_meet_date, execution_date=20211223T163743, start_date=20211223T163746, end_date=20211223T163747
[2021-12-23 16:37:47,085] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-23 16:37:47,114] {local_task_job.py:151} INFO - Task exited with return code 0
